src/projects/atlantic-wave-sdx.js

```diff
@@ -55,10 +56,15 @@ export const AtlanticWaveSdx = () => {
 
       <Subheading>Demo/Office Hours</Subheading>
 
-      <Paragraph>
-      The goal of this demo is to showcase the benefits and challenges of deploying In-band Network 
+      <Paragraph><span className="bold">Title:</span> In-Band Network Telemetry @ AmLight
```

Comment:
it would be simpler to use a `<strong>` tag here instead of extending/adding complexity to the `Paragraph` component